pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -140,7 +140,7 @@ func (r *ReconcileComplianceScan) Reconcile(request reconcile.Request) (reconcil
 }
 
 func (r *ReconcileComplianceScan) phasePendingHandler(instance *compv1alpha1.ComplianceScan, logger logr.Logger) (reconcile.Result, error) {
-	logger.Info("Phase: Pending", "ComplianceScan", instance.ObjectMeta.Name)
+	logger.Info("Phase: Pending", "ComplianceScan.Name", instance.ObjectMeta.Name)
```

Comment:
we're already logging the scan. That's done with the `log.WithValues` from line 100